SourceX/DiabloUI/diabloui.cpp

```diff
@@ -424,8 +424,10 @@ void InitFont()
 		return;
 	}
 
+#ifndef USE_SDL1

```

Comment:
This doesn't appear to be needed for the other SDL1 target, is this an Amiga specific issue?

Reply:
Correct, can be removed.

Reply:
If that's the case, why does it not link properly on SDL1.2? According to the documentation this should work on SDL1.2 SDL_ttf...

Reply:
Links here.